agent/xds/server.go

```diff
@@ -141,6 +150,36 @@ type Server struct {
 	AuthCheckFrequency time.Duration
 
 	DisableV2Protocol bool
+
```

Comment:
Am I correct in assuming that there can only be one Server instance per host? Do we run multiple servers on different ports or anything like that?

Reply:
There is exactly one xDS server per consul agent.